hieradata_aws/class/integration/db_admin.yaml

```diff
@@ -88,17 +88,6 @@ govuk_env_sync::tasks:
     temppath: "/tmp/content-register_production"
     url: "govuk-integration-database-backups"
     path: "postgresql-backend"
-  "push_content_audit_tool_production_integration_daily":
-    ensure: "present"
```

Comment:
Does removing this part mean that the relevant resources will be removed, or does `ensure` here need changing to absent first?